src/info_page.cpp

```diff
@@ -371,14 +409,20 @@ M2kInfoPage::~M2kInfoPage()
 
 }
 
+void M2kInfoPage::getDeviceInfo()
+{
+	InfoPage::getDeviceInfo();
+	//refreshTemperature();
```

Comment:
is this needed?

Reply:
This is a stub for future implementation of temperature based calibration - you can only do this on the M2K, not other boards